src/main/scala/com/mesosphere/cosmos/CosmosIntegrationTestServer.scala

```diff
@@ -66,19 +68,12 @@ final class CosmosIntegrationTestServer(
 
     val java = javaHome
       .map(_ + "/bin/java")
-      .orElse(systemProperty("java.home").map(jre => s"$jre/bin/java"))
+      .orElse(loadSystemProperty("java.home").map(jre => s"$jre/bin/java"))
       .getOrElse("java")
 
-    val dcosUri = systemProperty("com.mesosphere.cosmos.dcosUri").get
+    val dcosUri = loadDcosUriSystemProperty()
     val propertiesMap = additionalProperties.map { testProperty =>
-      testProperty -> systemProperty(testProperty.propertyName).get
-    }
-
-    val uriKey = "uri"
-    setClientProperty("CosmosClient", uriKey, "http://localhost:7070")
-
-    propertiesMap.foreach { case (testProperty, value) =>
-      setClientProperty(testProperty.clientName, testProperty.name, value)
```

Comment:
This is not needed. The integration shouldn't be required to know how Cosmos was executed. If they know how Cosmos was exec then it makes it difficult to make them re-targetable.